### PR TITLE
Release v0.1

### PR DESCRIPTION
**Pull request by phillipahereza**
### Added
- Improve HURUmap to use data releases
- Improve Docker
- Decouple `hurumap.dashboard`
- Added new Wazimap logo for dark backgrounds
- Multiple Chart Qualifiers Support

### Changed
- Hide Empty sections on the profile page
- Rename `wazi-logo` to `wazi-logo-white`
- Replace sourceAFRICA with openAFRICA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,5 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
Comment (kilemensi) [inline CHANGELOG.md]: Have we dropped semantic versioning?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,5 +2,17 @@
...
+- Improve HURUmap to use data releases
+- Improve Docker
+- Decouple `hurumap.dashboard`
+- Added new Wazimap logo for dark backgrounds
+- Multiple Chart Qualifiers Support
+
+### Changed
+- Hide Empty sections on the profile page
+- Rename `wazi-logo` to `wazi-logo-white`
+- Replace sourceAFRICA with openAFRICA
```
Comment (kilemensi) [inline CHANGELOG.md]: Have the docs/README been updated to show how to use data releases, multiple chart qualifiers and how to hide empty sections at least?

**Comment by phillipahereza** [comment]
@kilemensi ready

**Comment by phillipahereza** [comment]
@kilemensi changes have been made and I've also created a pre-release for this. What's the next step?

**Comment by kilemensi** [comment]
I think that's it for this PR (deploying new docker image etc. will come after merging).

**Pull request merged**